interface_service/src/main.py

```diff
@@ -1,6 +1,7 @@
 """Main file for the FastAPI application."""
 from typing import Set
 from fastapi import FastAPI
+from fastapi.responses import JSONResponse
```

Comment:
Is this necessary to import it if it is not used?

Reply:
It's not, looks like a leftover from when I was playing around with FastAPI's JSON serialization.